bedrock/releasenotes/models.py

```diff
@@ -149,7 +149,7 @@ class ProductRelease(models.Model):
     CHANNELS = ('Nightly', 'Aurora', 'Beta', 'Release', 'ESR')
     PRODUCTS = ('Firefox', 'Firefox for Android',
                 'Firefox Extended Support Release', 'Firefox OS',
-                'Thunderbird', 'Firefox for iOS')
+                'Firefox for iOS')
```

Comment:
I took Thunderbird out of the products list. Is that okay?

Reply:
I'm not quite sure where this is displayed, but if this removes it from some mozilla.org page headers, I'd prefer if we could keep it in and have it link to Thunderbird.net. @Sancus probaby has a better idea what this is for.

Reply:
afaik, Thunderbird is not linked to by any other part of the mozilla.org site these days.

Reply:
Very well then, if this is the last link then it seems reasonable to remove.

Reply:
I believe this is connected to the database some how. Probably needs an okay from @pmac.

Reply:
Please don't remove this. It has to do with importing releases from our release notes server which would fail since this is used as a list of allowed values for a DB field. We'll have to remove this after Thunderbird notes are removed from said server (Nucleus).